src/integrationTest/resources/sql/insert_org_assignment_records_to_delete.sql

```diff
@@ -0,0 +1,19 @@
+DELETE FROM role_assignment_history;
+DELETE FROM role_assignment_request;
+DELETE FROM role_assignment;
+DELETE FROM actor_cache_control;
+
```

Comment:
Why you create  this file if there is no use ?

Reply:
Removed